alpha/alpha-core/src/main/java/org/apache/servicecomb/pack/alpha/core/CompositeOmegaCallback.java

```diff
@@ -54,6 +56,11 @@ public void compensate(TxEvent event) {
 
     try {
       omegaCallback.compensate(event);
+    } catch (CompensateConnectException e) {
```

Comment:
I think we just need to check the CompensateAskFailedException here.
, as the other exception catch part just moves the callback connection.
 I doubt CompensateAskFailedException is a typo, the right one should be CompensateAckFailedException.

Reply:
Yes, I will rename CompensateAskFailedException to CompensateAckFailedException.

Because CallbackContext does not throw an exception, when the connection is closed we need to throw a CompensateConnectException.

In addition, CompensateConnectException and CompensateAckFailedException have different compensation failure strategies.

- CompensateConnectException: Immediately call another Omega instance
- CompensateAckFailedException: Retry until the max compRetries is reached before calling another Omega instance